Assets/Shopify/UIToolkit/Test/Integration/TestRemoteImageLoaderIntegration.cs

```diff
@@ -0,0 +1,36 @@
+#if !UNITY_IOS
+namespace Shopify.UIToolkit.Test.Integration {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using UnityEngine.TestTools;
+    using System.Collections;
+    using Shopify.Unity.SDK;
+
+    [IntegrationTest.DynamicTest("UIToolkitIntegrationTests")]
+    [IntegrationTest.Timeout(10)]
```

Comment:
I think @mikkoh removed this... can you rebase on master to make sure this still compiles without the test tools?